paddle/fluid/operators/sgd_op.cu

```diff
@@ -97,13 +96,15 @@ class SGDOpCUDAKernel : public framework::OpKernel<T> {
       auto* in_data = in_value.data<T>();
       auto* out_data = param_out->data<T>();
 
-      const int block_size = 256;
-      dim3 threads(block_size, 1);
-      dim3 grid(1, in_rows.size());
```

Comment:
Is the new way faster?

Reply:
This fix may not make this Op fast, it only makes sure this Op can deal with any size of input.